arctic/store/version_store.py

```diff
@@ -66,6 +70,10 @@ def _ensure_index(self):
         collection.versions.create_index([('symbol', pymongo.ASCENDING), ('version', pymongo.DESCENDING)], unique=True,
                                          background=True)
         collection.version_nums.create_index('symbol', unique=True, background=True)
+        collection.metadata.create_index([('symbol', pymongo.ASCENDING), ('_start_time', pymongo.DESCENDING)],
```

Comment:
Can you add a unique constraint on `(symbol, start)`? This would prevent multiple entries being stored for the same symbol because of a retry or similar.